articles/how-to-guides/create-an-application-template/create-an-application-template.md

```diff
@@ -2,3 +2,83 @@
 uid: how-to-guides.create-an-application-template
 ---
 # Create an Application Template
+
+Do you find yourself creating new applications in Intent Architect by choosing a very simple application template and customizing it by installing lots of modules?
+Working in a small team or organization can become daunting when someone else needs to create a new application and are not sure what the configuration settings should be.
+
+In this how-to we will explain how to setup your own team or organizational application template list which can run side-by-side to the official application template list and allows developers not only to create their own applications with ease, but they can easily customize it without having too much in-depth knowledge upon creation.
+
+In Intent Architect, choose to create a new application and select the `Intent Application Template Builder`.
+Give it the name of `MyApplicationTemplates`.
+
+![Application Template Builder Creation](images/my-app-templates-creation.png)
+
+When clicking on `Next` you will be presented with the following page. There is nothing to customize here so you can click on `Create` to begin your own application templates.
+
+![Application Template Builder Customization](images/my-app-templates-customize.png)
+
+Once the installation dialog is complete, click on `Close`.
+
+![Installation dialog](images/installation-dialog.png)
+
+Open the `App Templates` designer and click on `Create new pacakge`.
+
+![Create App Template Package](images/create-app-templates-package.png)
+
+If you find that the package is highlighted in red, click on it and complete the following fields:
+
+![Application Template Package Settings](images/app-template-pacakge-settings.png)
+
+| Name | Description |
+|-|-|
+| Version | You can give it a default value of `1.0.0`. **It needs to have 3 version components.** |
```

Comment:
Perhaps rather state it needs to be a _semver_ string with a link to the semver spec.